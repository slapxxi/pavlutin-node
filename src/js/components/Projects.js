import React from 'react';
import Project from './Project';
import { Container } from './Projects.styled';

function Projects({ projects }) {
  if (projects.length === 0) {
    return <p>There are no projects yet.</p>;
  }
  return (
    <Container>
      {projects.map(p => <Project key={p.id} project={p} />)}
    </Container>
  );
}

export default Projects;
