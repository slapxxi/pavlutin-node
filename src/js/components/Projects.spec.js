import React from 'react';
import render from 'react-test-renderer';
import Projects from './Projects';

const projects = [
  { id: 0, title: 'First Projects' },
  { id: 1, title: 'Second Project' },
];

it('renders projects', () => {
  const tree = render.create(<Projects projects={projects} />);
  expect(tree.toJSON()).toMatchSnapshot();
});
