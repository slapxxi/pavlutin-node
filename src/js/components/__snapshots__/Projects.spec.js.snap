// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders projects 1`] = `
<ul
  className="hqGPSu"
>
  <li
    className="gEoxXY"
  >
    <h1>
      First Projects
    </h1>
  </li>
  <li
    className="gEoxXY"
  >
    <h1>
      Second Project
    </h1>
  </li>
</ul>
`;
